lib/scrape.js

```diff
@@ -184,6 +185,33 @@ async function findOrganization({
   return null
 }
 
+async function fetchSEtag(wikidata_id) {
+  const url = wdk.getEntities({
+    ids: [wikidata_id],
+    props: ['claims'],
+  })
+  const page = await fetch(url)
+  let data = await page.json()
+  data = data.entities[wikidata_id]
+  //P1482 is Stack Exchange tag
+  if ('P1482' in data.claims) {
+    const url = wdk.simplify.claim(data.claims.P1482[0])
+    const response = await fetch(url)
+    const html = cheerio.load(await response.text())
```

Comment:
Better place that `await` in a variable first before apply it to a function argument.